packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## `6.26.1`
```

Comment:
I believe we don't want to lock the changelog down to a specific version (to prevent having to merge PRs in a specific order).

Could we change this to `## Recent Changes` for now?